ge_worker/bonuses/bonus_processor.py

```diff
@@ -0,0 +1,141 @@
+import os
+import sys
+
+from datetime import datetime
+from datetime import timedelta
+from flask import g
+from logging import Logger
+
+parent_dir = os.path.abspath(os.path.join(os.getcwd(), "../app"))
```

Comment:
`os.getcwd()`

is this dangerous? Should we get the script path instead?

Reply:
This implementation is done with conventions of the Sempo build in mind. I was experimenting with a different flask project [see setup script](https://github.com/PhilipWafula/faulu-apis-demo/blob/master/setup.sh) and I decided to use the approach we had discussed while in Kilifi.
I'd argue that adding the root folder of the whole application to the python path would help us avoid the numerous occurrences of this in the code. It is really irksome having to add this every time I need to access that server package.
@nolash what do you think ?

Reply:
>  It is really irksome having to add this every time I need to access that server package.

Yeah it seems like bad design. I concur.